keyboards/kprepublic/bm40hsrgb/keymaps/dan/config.h

```diff
@@ -0,0 +1,4 @@
+#pragma once
```

Comment:
Could you add a GPL2+ compatible license header here?  All of the source files need to have a license explicitly listed.

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```

Reply:
Hi I have submitted another commit regarding this. There are some shift arounds on keymap but not a major one ( just added tap dance to a numpad layer to access * and / from a single key ). Thanks!